core/edit_text_file.py

```diff
@@ -9,25 +9,35 @@
 
 mod = Module()
 ctx = Context()
+mod.list(
+    "edit_file",
+    desc="Absolute paths to frequently edited files (.talon-list, csv, etc)",
+)
+
+_edit_files = {
+    "alphabet": os.path.join(REPO_DIR, "core\\keys\\letter.talon-list"),
```

Comment:
Should either use `pathlib` and the `/` operator or `os.sep` or additional arguments rather than `\`.